src/kubeseal_auto/core/kubeseal.py

```
@@ -0,0 +1,303 @@
+        detached_mode: Whether operating without direct cluster access.
+        binary: Path to the kubeseal binary.
+        certificate: Path to the certificate file (detached mode only).
+        cluster: Cluster instance for cluster operations.
+        controller_name: Name of the SealedSecrets controller.
+        controller_namespace: Namespace of the SealedSecrets controller.
+        current_context_name: Current Kubernetes context name.
+        namespaces_list: List of available namespaces.
+
+    """
+
+    def __init__(self, select_context: bool, certificate: str | None = None) -> None:
```

Ruff (0.14.7)

Boolean-typed positional argument in function definition

(FBT001)